script/Main.lua

```diff
@@ -28,6 +28,7 @@ function InitSystem()
 
         GlobalRestrict.On()
 
+        dofile('./script/Config/Version.lua')
```

Comment:
We can add a method to the Engine that will return this version string

Reply:
I am overwriting this string from the GitHub action, do you know how that would work for the engine?

Reply:
Just replace it somewhere inside the engine files i guess? If yes go ahead :D

Reply:
@HaronK I think it might actually be a better idea to have both a version baked into the Lua code here, and a separate copy of the same version compiled into the engine binary (passing it into `build.sh`). That way, we can check at runtime if the versions match in case someone forgot to update their `ltr.exe`.

I quite like the fact the GitHub action parses the Cargo.toml and so we only have one canonical version to update after we do a major release.

Reply:
Ok, I can add this check in the next PR